lambdas/get-addresses/tsconfig.json

```diff
@@ -3,14 +3,16 @@
         "target": "es2020",
         "strict": true,
         "preserveConstEnums": true,
-        "noEmit": true,
-        "sourceMap": false,
+        // "noEmit": true, // emit for debugger testing purposes.
```

Comment:
Does `tsconfig.json` allow comments inside it - normal JSON errors if you try that.